### PR TITLE
doc: model loading page

### PR DESCRIPTION
**Pull request by ThibaultFy**
Signed-off-by: ThibaultFy <50656860+ThibaultFy@users.noreply.github.com>

## Related issue

`#` followed by the number of the issue

## Please check if the PR fulfills these requirements

- [ ] If the feature has an impact on the user experience, the [changelog](https://github.com/substra/substrafl/blob/main/CHANGELOG.md) has been updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The commit message follows the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+### Added
+
+- doc on the model loading page ()
```
Comment (RomainGoussault) [inline CHANGELOG.md:12]: Why the "()"? It should be the PR number?

File: docs/api/model_loading.rst [hunk 2]
```diff
@@ -1,4 +1,9 @@
 Model Loading
 =============
 
+To use a model that has been trained on substra, you need to first download the files with the ``download_algo_files`` function.
+These files are used to re-instantiate and load all variables to retrieve the user's Algo in its wanted state by the ``load_algo`` function.
+
+An example on "How to download a model" is available in the Substrafl FedAvg on MNIST dataset example.
+
```
Comment (RomainGoussault) [inline docs/api/model_loading.rst:8]:
```suggestion
To use a model that has been trained on Substra, you need to first download the files with the ``download_algo_files`` function.
These files are used to re-instantiate and load all variables to retrieve the Substra Algo in its wanted state by the ``load_algo`` function.

An example on how to download a model is available in the MNIST Substrafl FedAvg example.

```

File: CHANGELOG.md (hunk 1 above)
Comment (ThibaultFy) [reply CHANGELOG.md]: Thanks, I wrote the changelog before having the number of the pr and forgot to update :)

**Review by RomainGoussault — APPROVED** [review]
Thanks for the PR!

File: docs/api/model_loading.rst [hunk 3]
```diff
@@ -1,4 +1,9 @@
 Model Loading
 =============
 
+To use a model that has been trained on Substra, you need to first download the files with the ``download_algo_files`` function.
+These files are used to re-instantiate and load all variables to retrieve the Substra Algo in its wanted state by the ``load_algo`` function.
+
+An example on how to download a model is available in the MNIST Substrafl FedAvg example.
```
Comment (guilhem-barthes) [inline docs/api/model_loading.rst:7]: Maybe add a link to the example:

```suggestion
An example on how to download a model is available in the :doc:`MNIST Substrafl FedAvg example </substrafl_doc/examples/get_started/plot_substrafl_torch_fedavg>`.
```

**Review by guilhem-barthes — APPROVED** [review]

**Pull request merged**